guides/common/assembly_administering-hosts.adoc

```diff
@@ -24,7 +24,10 @@ include::modules/proc_changing-the-environment-of-a-host.adoc[leveloffset=+1]
 
 include::modules/proc_changing-the-managed-status-of-a-host.adoc[leveloffset=+1]
 
-include::modules/proc_enabling-tracer-on-a-host.adoc[leveloffset=+1]
+:parent-context: {context}
+:context: administering-hosts
```

Comment:
Is there any particular reason to change the context for this procedure?
Unless it's absolutely necessary to change the context, I would avoid it. Generally, it prevents problems with broken links in the web UI.

Reply:
@Lennonka The two original procedures that I've merged into the one you reviewed still need to be kept in both places, because the following sections rely on them. For this reason, not changing context doesn't work due to duplicate IDs resulting from two inclusions of the module.